processor/agentmetricsprocessor/agentmetricsprocessor.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 OpenTelemetry Authors
```

Comment:
Shouldn't it be Google Inc?

Reply:
Added licensecheck & test steps to CI as well